T: 0.1
N: 100

constraints:
  v_max: 1.3
  v_min: -1
  v_ref: 1
  delta_max: 0.4
  delta_min: -0.4
  x_min: 0.0
  x_max: 15.0
  y_min: 0.0
  y_max: 15.0

costs:
  y_cost: 2
  x_cost: 2
  yaw_cost: 0.5
  v_cost: 1 
  steer_cost: 0.25
  delta_v_cost: 0.25
  delta_steer_cost: 0.5